libs/cuda_support/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+if (NOT TARGET hpx::cuda)
+  message(FATAL_ERROR "Need cuda")
+endif()
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Default location is $HPX_ROOT/libs/cuda_support/include
+set(cuda_support_headers
+  hpx/cuda_support/cuda_future_helper.hpp  
+)
+
+# Default location is $HPX_ROOT/libs/cuda_support/include_compatibility
+set(cuda_support_compat_headers)
```

Comment:
Please generate forwarding headers to retain compatibility with existing code and add those headers here.